aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -222,10 +222,13 @@ module std::features {
     /// Lifetime: transient
     const APTOS_UNIQUE_IDENTIFIERS: u64 = 23;
 
-    public fun get_auids(): u64 { APTOS_UNIQUE_IDENTIFIERS }
+    public fun get_auids(): u64 {
+        // Deployed to production, and disabling deprecated.
+        error::invalid_argument(EINVALID_FEATURE)
```

Comment:
Probably permission_denied or something like this would be better? With a new error code which is like EFEATURE_CANNOT_BE_DISABLED or something like that